docs-src/references/dynamic-configuration.md

```diff
@@ -0,0 +1,209 @@
+
+  ```yaml
+  matching.numTaskqueueReadPartitions: # Number of Task Queue partitions for read operations
+  - constraints: {namespace: "namespace1", taskQueueName: "tq"}  # Applies to the "tq" Task Queue for both Workflows and Activities.
+    value: 8 # The default value for this key is 4. Task Queues that need to support high traffic require higher number of partitions. Set these values in accordance to your poller count. 
+  - constraints: {namespace: "namespace1", taskQueueName: "other-tq", taskType: "Activity"} # Applies to the "other_tq" Task Queue for Activities specifically.
+    value: 20 
+  - constraints: {namespace: "namespace2"}  # Applies to all task queues in "namespace2".
+    value: 10
+  - constraints: {}  # Applies to all other task queues in "namespace1" and all other Namespaces.
+    value: 16
+  matching.numTaskqueueWritePartitions:  # Number of Task Queue partitions for write operations
+  - constraints: {namespace: "namespace1", taskQueueName: "tq"}  # Applies to the "tq" Task Queue for both Workflows and Activities.
+    value: 8 # The default value for this key is 4. Task Queues that need to support high traffic require higher number of partitions. Set these values in accordance to your poller count. 
+  - constraints: {namespace: "namespace1", taskQueueName: "other-tq", taskType: "Activity"} # Applies to the "other_tq" Task Queue for Activities specifically.
+    value: 20 
+  - constraints: {namespace: "namespace2"}  # Applies to all task queues in "namespace2".
+    value: 10
+  - constraints: {}  # Applies to all other task queues in "namespace1" and all other Namespaces.
+    value: 16
+  ```
+
+<!-- Note that the values set with most constraints take priority over values that are set with fewer constraints, regardless of the order in which they are set in the dynamic configuration key. -->
+
+For more examples on how dynamic configuration is set, see:
+
+- [docker-compose](https://github.com/temporalio/docker-compose/tree/main/dynamicconfig)
+- [samples-server](https://github.com/temporalio/samples-server/blob/main/tls/config/dynamicconfig/development.yaml)
+
+## Commonly used dynamic configuration keys
+
+The following table lists commonly used dynamic configuration keys that can be used for rate limiting requests to the Temporal Cluster.
+
+Note that the dynamic configuration key setting is optional. If you choose to update these values for your Temporal Cluster, ensure that you are provisioning enough resources to handle the load.
+
+All values listed here are for Temporal server v1.20 <!--Update to 1.21 when released-->.
+Check [server release notes](https://github.com/temporalio/temporal/releases) to verify any potential breaking changes when upgrading your versions.
+
+### Service-level RPS limits
+
+The Rate Per Second (RPS) dynamic configuration keys set the rate at which requests can be made to each service in your Cluster.
```

Comment:
```suggestion
The Requests Per Second (RPS) dynamic configuration keys set the rate at which requests can be made to each service in your Cluster.
```

"rate" is already "something per second" (or unit of time), a "rate per second" would be like an acceleration :)

Reply:
lol, corrected (ty)